docs/topics/api/overview.rst

```diff
@@ -27,6 +27,12 @@ There are some common API responses that you can expect to receive at times.
 
 .. http:get:: /api/v3/...
 
+    :statuscode 200: Success.
+    :statuscode 201: Creation successful.
+    :statuscode 202: The request has been accepted for processing.
+    :statuscode 204: Success (no content is returned).
+    :statuscode 400:
+        There was an error with the parameters sent with this request.
```

Comment:
I'm not sure if I like that indentation but for now I don't neccessarily dislike it, just thinking out loud :D